tutorials/nlp/Relation_Extraction-BioMegatron.ipynb

```diff
@@ -0,0 +1,601 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "uRLPr0TnIAHO"
+   },
+   "outputs": [],
+   "source": [
+    "BRANCH = 'main'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "o_0K1lsW1dj9"
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "\"\"\"\n",
+    "# If you're using Google Colab and not running locally, run this cell\n",
+    "\n",
+    "# install NeMo\n",
+    "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@{BRANCH}#egg=nemo_toolkit[nlp]\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# If you're not using Colab, you might need to upgrade jupyter notebook to avoid the following error:\n",
+    "# 'ImportError: IProgress not found. Please update jupyter and ipywidgets.'\n",
+    "\n",
+    "! pip install ipywidgets\n",
+    "! jupyter nbextension enable --py widgetsnbextension\n",
+    "\n",
+    "# Please restart the kernel after running this cell"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "dzqD2WDFOIN-"
+   },
+   "outputs": [],
+   "source": [
+    "from nemo.collections import nlp as nemo_nlp\n",
+    "from nemo.utils.exp_manager import exp_manager\n",
+    "\n",
+    "import os\n",
+    "import wget \n",
+    "import torch\n",
+    "import pytorch_lightning as pl\n",
+    "from omegaconf import OmegaConf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "daYw_Xll2ZR9"
+   },
+   "source": [
+    "In this tutorial, we are going to describe how to finetune a BERT-like model based on [BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805) on [RE: Text mining chemical-protein interactions (CHEMPROT)](https://biocreative.bioinformatics.udel.edu/tasks/biocreative-vi/track-5/). "
+   ]
```

Comment:
Could you please add a longer textual description of why we're using BioMegatron - how it's different from the regular BERT, what data it was trained on, and etc. what are BioMegatron alternatives? Also, there is some text in the original notebooks describing the task that might be worth adding here.